megamek/src/megamek/common/verifier/TestAero.java

```diff
@@ -170,38 +170,40 @@ public static int slotsPerArc(Aero aero) {
     public static int maxBayDoors(Aero aero) {
         if (aero.hasETypeFlag(Entity.ETYPE_WARSHIP)) {
             return 8 + (int) Math.ceil(aero.getWeight() / 100000);
-        } else if (aero.hasETypeFlag(Entity.ETYPE_JUMPSHIP)) {
+        } else if (aero.hasETypeFlag(Entity.ETYPE_SPACE_STATION)) {
```

Comment:
Question: is there a rulebook reference for this? Obviously it's a bug, but just want to make sure we're using the right formula for the right stuff.

Reply:
`ADVANCED AEROSPACE UNIT 
MAXIMUM BAY DOORS TABLE
Advanced Unit Type Maximum Bay Doors
JumpShips 7 + [Unit Weight ÷ 50,000]*
Space Stations 8 + [Unit Weight ÷ 75,000]*
WarShips 8 + [Unit Weight ÷ 100,000]*`

- `Strategic Operations` p. 155